File: Lab2/app/src/main/java/com/example/lab2/fragment/OutputFragment.java
package com.example.lab2.fragment;

import android.graphics.Typeface;
import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import androidx.fragment.app.Fragment;

import com.example.lab2.ETextFont;
import com.example.lab2.R;


public class OutputFragment extends Fragment {

    private static final String ARG_TEXT = "text";
    private static final String ARG_FONT = "font";

    private String mText;
    private String mFont;

    public OutputFragment() {

    }

    public static OutputFragment newInstance(String text, String font) {
        OutputFragment fragment = new OutputFragment();
        Bundle args = new Bundle();
        args.putString(ARG_TEXT, text);
        args.putString(ARG_FONT, font);
        fragment.setArguments(args);
        return fragment;
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (getArguments() != null) {
            mText = getArguments().getString(ARG_TEXT);
            mFont = getArguments().getString(ARG_FONT);
        }
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        View view = inflater.inflate(R.layout.fragment_output, container, false);
        TextView textView = view.findViewById(R.id.infoMessage);
        ETextFont eTextFont = ETextFont.getTextFontByName(mFont);
        if (eTextFont != null) {
            Typeface typeface = ETextFont.getTextFontByName(mFont).getTypeface();
            textView.setText(mText);
            textView.setTextSize(22);
            textView.setTypeface(typeface);
        }
        return view;
    }
}

File: Lab3/app/src/main/java/com/example/lab3/fragment/OutputFragment.kt
package com.example.lab3.fragment

import android.os.Bundle
import androidx.fragment.app.Fragment
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import com.example.lab3.enumerable.ETextFont
import com.example.lab3.R

class OutputFragment : Fragment() {

    private var mText: String? = null
    private var mFont: String? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        arguments?.let {
            mText = it.getString(ARG_TEXT)
            mFont = it.getString(ARG_FONT)
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        val view = inflater.inflate(R.layout.fragment_output, container, false)
        val textView = view.findViewById<TextView>(R.id.input_text)
        mText?.let {
            mFont?.let {
                val typeface = ETextFont.getTextFontByName(mFont!!).typeface
                textView.text = mText
                textView.textSize = TEXT_SIZE
                textView.typeface = typeface
            }
        }
        return view
    }

    companion object {

        private const val TEXT_SIZE = 22f

        private const val ARG_TEXT = "text"
        private const val ARG_FONT = "font"

        @JvmStatic
        fun newInstance(text: String, font: String) =
            OutputFragment().apply {
                arguments = Bundle().apply {
                    putString(ARG_TEXT, text)
                    putString(ARG_FONT, font)
                }
            }
    }
}